### PR TITLE
v1.1.0 release

### PR DESCRIPTION
**Pull request by emmanuelmathot**
**Related Issue(s):** #

**Proposed Changes:**

1. Update version numbers to v1.1.0

**PR Checklist:**

- [X] This PR is made against the dev branch (all proposed changes except releases should be against dev, not master).
- [X] This PR has **no** breaking changes.
- [X] I have added my changes to the [CHANGELOG](https://github.com/radiantearth/stac-spec/blob/dev/CHANGELOG.md)
      **or** a CHANGELOG entry is not required.
- [ ] This PR affects the [STAC API spec](https://github.com/radiantearth/stac-api-spec),
      and I have opened issue/PR #XXX to track the change.

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -8,9 +8,9 @@ preferred-citation:
   type: standard
   title: "SpatioTemporal Asset Catalog (STAC) specification"
   abstract: "Making geospatial assets openly searchable and crawlable."
-  version: 1.1.0-beta.1
+  version: 1.1.0
   year: 2024
-  date-released: 2024-08-08
+  date-released: 2024-09-08
```
Comment (m-mohr) [inline CITATION.cff:13]:
```suggestion
  date-released: 2024-09-10
```
As we'll probably not release today

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,12 +6,21 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [v1.1.0] - 2024-09-08
```
Comment (m-mohr) [inline CHANGELOG.md:9]:
```suggestion
## [v1.1.0] - 2024-09-10
```

**Review by m-mohr — CHANGES_REQUESTED** [review]
We should probably release eo and raster first and also update the version numbers (removing the -beta.1) of those extensions?!

**Review by jbants — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -508,7 +517,7 @@ See the [milestone 0.4.0 in the issue tracker](https://github.com/radiantearth/s
 
 Thanks @hgs-msmith, @matthewhanson, @hgs-trutherford, @rouault, @joshfix, @alkamin, @hemphillda, @jeffnaus  and @fredliporace for contributing to the spec directly, and to [everyone](https://github.com/opengeospatial/wfs3hackathon/blob/master/notes/introductions.md#participants) who participated in the [Ft Collins sprint](https://github.com/radiantearth/community-sprints/tree/master/03072018-ft-collins-co) and brought great ideas.
 
-[Unreleased]: <https://github.com/radiantearth/stac-spec/compare/master...dev>
+[v1.1.0]: <https://github.com/radiantearth/stac-spec/compare/v1.1.0-beta.1...v1.1.0>
```
Comment (m-mohr) [inline CHANGELOG.md:520]:
```suggestion
[Unreleased]: <https://github.com/radiantearth/stac-spec/compare/master...dev>
[v1.1.0]: <https://github.com/radiantearth/stac-spec/compare/v1.1.0-beta.1...v1.1.0>
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -6,12 +6,21 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [v1.1.0] - 2024-09-10
```
Comment (m-mohr) [inline CHANGELOG.md:9]:
```suggestion
## [Unreleased]

## [v1.1.0] - 2024-09-10
```

**Pull request merged**